lib/std/json/write_stream.zig

```diff
@@ -197,6 +197,34 @@ pub fn WriteStream(comptime OutStream: type, comptime max_depth: usize) type {
             try self.stream.writeByte('"');
         }
 
+        /// Writes the complete json into the output stream
+        pub fn writeJson(self: *Self, json: std.json.Value) std.os.WriteError!void {
```

Comment:
```suggestion
        pub fn emitJson(self: *Self, json: std.json.Value) Stream.Error!void {
```

Here I'm suggesting:
 * `emitJson` to match the other public functions in this API
 * correcting the error set